lib/Chunk.js

```diff
@@ -21,6 +21,27 @@ const sortByIdentifier = (a, b) => {
 	return 0;
 };
 
+const getFrozenArray = set => Object.freeze(Array.from(this));
+
+const getModulesIdent = set => {
+	set.sort();
+	let str = "";
+	set.forEach(m => {
+		str += m.identifier() + "#";
+	});
+	return str;
+};
+
+const getArray = set => Array.from(set);
```

Comment:
move to `lib/util/toArray.js` && `getArray` =>`toArray` ?

Reply:
I don't think a separate module make sense for a single line of code. This increases code size, decreases performance and readablility.

Reply:
Yeah, it isn't _really_ needed here, especially after the last refactor :)